console/v1/types_console_external_log_links.go

```diff
@@ -0,0 +1,40 @@
+package v1
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+// +genclient
+// +genclient:nonNamespaced
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// ConsoleExternalLogLink is an extension for customizing OpenShift web console pod log links.
+type ConsoleExternalLogLink struct {
+	metav1.TypeMeta `json:",inline"`
+	// Standard object's metadata.
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+	Spec              ConsoleExternalLogLinkSpec `json:"spec"`
+}
+
+// ConsoleExternalLogLinkSpec is the desired console pod log link configuration.
```

Comment:
I would state the link will appear on the logs tab of the pod details page.

Reply:
Something like:`ConsoleExternalLogLinkSpec is the desired log link configuration.  The log link will appear on the logs tab of the pod detail page.`